inspector/templates/base.html

```diff
@@ -1,59 +1,69 @@
 <html>
-  <head>
+<head>
```

Comment:
@AngeloD2022 Seems like there's some unnecessary formatting/indenting changes in this PR like this, would you mind reverting those?